### ISSUE TITLE
Delete/ remove 'Deleted' & 'Outbox'

### ISSUE THREAD

**Issue by xpseudonym** [cleidigh/Localfolder-TB#67]
Is there any way to remove the 'Deleted' & 'Outbox' - their checkboxes are dimmed out and uneditable on a new 'Local Folders' creation dialogue?

All I'm wanting is a hierarchy to hang some global search folders under - if I delete them, they just go rather than getting transferred to the 'Deleted' folder and obviously the 'Outbox' folder is irrelevant.

Thanks

**Comment by cleidigh** [comment]
@xpseudonym 
As a pseudo account these are created by Thunderbird automatically. These are basically expected to be there. Cannot say if 102 is different.
@cleidigh

**Comment by xpseudonym** [comment]
Thanks @cleidigh So, that's a function of TB and not the extension?  And if so, would presumably require hacking on TB somewhere?

**Comment by cleidigh** [comment]
@xpseudonym 
TB still wants both to exist.
I had to remind myself what I found three years ago when I tried making these optional. I also checked if anything has changed in 102, nope.

First if you manually delete them outside TB, they will be restored on startup.
Second, if you delete messages in any local folder, it should go to the trash of the local folder.

Thought about hiding, but TB would have to do that.
@cleidigh
